profile.css

```diff
@@ -5,31 +5,162 @@ body {
 
 }
 
+.container {
+    width: 1200px;
+}
+
 .col-4 {
-    /*position: relative;*/
+    position: relative;
     float: left;
-    width: 33.33%;
-    height: 400px;
     padding: 50px 30px;
-    /*border: 1px solid #FFF;*/
 }
 
 .col-8 {
     float: left;
     width: 66.66%;
-    height: 400px;
+    /*height: 400px;*/
     padding: 50px 10px;
     /*border: 1px solid #FFF;*/
 }
 
 .profile-image {
     border-radius: 50%;
+    height: 250px;
+    width: 250px;
+    margin-top: 20px;
+    margin-left: 45px;
+}
+
+.icons {
+    clear: both;
+    text-align: center; /* How to center inline elements*/
+    padding-top: 70px;
+}
+
+.page-link {
+    margin-top: 75px;
+}
+
+h2 {
+    margin-bottom: 7px;
+}
+
+p {
+    font-size: 1.3rem;
+}
+
+.contact{
+    margin: 0;
 }
 
-.page-links-container-profile {
-    /* Vertically and horizontally centre div */
-    position: absolute;
-    top: 90%;
-    left: 50%;
-    transform: translate(-50%, -50%);
+/****************************
+Media Queries
+/****************************/
+
+@media screen and (max-width: 1190px) {
+
+    .container {
+        height: 1100px;
+        width: 80%;
+    }
+
+    h1 {
+        margin-bottom: 0;
+    }
+
+    h2 {
+        text-align: center;
+    }
+
+    .col-4 {
+        float: none;
+        text-align: center;
+    }
+
+    .profile-image {
+        margin-top: 0;
+        margin-left: 0;
+    }
+
+    .col-8 {
+        float: none;
+        margin: 0 auto;
+        padding: 0;
+        width: 80%;
+    }
+
+    p {
+        text-align: center;
+    }
+
+    .icons {
+        padding-top: 0;
+    }
+
+}
+
+@media screen and (max-width: 770px) {
+
+    .page-link {
+        margin-top: 45px;
+    }
+
+}
+
+@media screen and (max-width: 770px) {
```

Comment:
Media query condition is the same as the one above (770px).

Reply:
Good spot. Have combined them.